source/simulator/parameters.cc

```diff
@@ -451,6 +451,27 @@ namespace aspect
                            "\\cite{KHB12}.");
       }
       prm.leave_subsection ();
+      prm.enter_subsection ("GMG parameters");
```

Comment:
Would it make sense to put this setting outside of GMG because it is also feasible to implement IDR if we are using the matrix-based solver? This doesn't need to be implemented right now of course.

Reply:
I agree, this should be in the general Stokes solver section, even if at the moment it only works for GMG.

Reply:
Should I add an Assert if this is called without GMG? This issue with not using this for AMG is that preconditioner requires a flexible Krylov method.